tests/outputs/u_institutions/institutions.yaml

```diff
@@ -0,0 +1,38 @@
+"columbiau":
```

Comment:
this is yaml so I don't think these things should be in quotes.

Reply:
done!